articles/industrial-iot/tutorial-configure-industrial-iot-components.md

```diff
@@ -48,25 +48,21 @@ output of deployment script or reset the password
 * IoT Hub → IoT Edge → \<DEVICE> → Set Modules → OpcPublisher (for standalone OPC Publisher operation only)
 
 
-## Configuration options
+## OPC Publisher Configuration options
 
 |Configuration Option (shorthand/full name)    |    Description   |
 |----------------------------------------------|------------------|
 pf/publishfile |The filename to configure the nodes to publish. If this option is specified, it puts OPC Publisher into standalone mode.
+s/site |The site OPC Publisher is assigned to.
 lf/logfile |The filename of the logfile to use.
 ll/loglevel |The log level to use (allowed: fatal, error, warn, info, debug, verbose).
-me/messageencoding |The messaging encoding for outgoing messages allowed values: Json, Uadp
```

Comment:
Why do we want to remove these these options? (me, mm, fm, di, hb)

Reply:
this list of commands is for orchestrated mode only.
According to this page [publisher-commandline](https://github.com/Azure/Industrial-IoT/blob/main/docs/modules/publisher-commandline.md) these options are available only in standalone mode